src/Speck.js

```diff
@@ -134,13 +134,32 @@ class Speck {
     return JSON.parse(JSON.stringify(rawData));
   }
 
+  toJSONContext(context) {
```

Comment:
The point is, what ```user.getContext('astronaut')``` will return ?  
If we go with this solution, shouldn't ```user.getContext('astronaut').validate('login')```  be valid ?